transforms/property-to-computed/__testfixtures__/import.input.js

```diff
@@ -0,0 +1,5 @@
+const Person = EmberObject.extend({
+  fullName: function() {
```

Comment:
Is it `() =>` a case to consider or in AST world it will get covered by already existing code?

Reply:
hmm im not sure, but i also dont think it would be vaild syntax to do `fullName(){ ... }.property()` in ember. I will investigate.

Reply:
The use case I was having in mind is
```
const Person = EmberObject.extend({
  fullName:() => {
      return `${this.firstName} ${this.lastName}`;
  }.property('firstName', 'lastName'),
});

Reply:
In that case its invalid syntax for ember and would not work.